godot-fmt/benches/gdext_tests.rs

```diff
@@ -0,0 +1,34 @@
+use std::{path::PathBuf, str::FromStr};

```

Comment:
Since this is a benchmark and not a test, would it make sense to name the file `gdext_bench.rs`?